src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/_help.py

```diff
@@ -7,5 +7,160 @@
 
 helps['backup'] = """
             type: group
-            short-summary: Backup cmdlets.
+            short-summary: Backup commands.
```

Comment:
As Derek pointed out, the consistent text really would be "Commands to manage Azure Backups."  (Commands to manage... being the important phrase)